docs/en/01_basic-configuration.md

```diff
@@ -125,49 +134,49 @@ Project\ORM\YourDataObject:
 Then implement the StaticPublishingTrigger interface on your DataObject:
 
 ```php
+namespace App\Models;
```

Comment:
Doesn't match the namespace in the YAML above (probably update the YAML to match this rather than the other way around)

Reply:
Updated